pkg/targets/reconciler/azureeventhubstarget/adapter.go

```diff
@@ -82,6 +83,10 @@ func makeTargetAdapterKService(target *v1alpha1.AzureEventHubsTarget, cfg *adapt
 		})
 	}
 
+	if target.Spec.DiscardCEContext {
+		envs = append(envs, corev1.EnvVar{Name: envDiscardCECtx, Value: "true"})
```

Comment:
nit: the conditional isn't even necessary. The user could have set this value explicitly to `false` in their spec, and expect this value to appear in the env vars of the adapter.

https://github.com/triggermesh/triggermesh/blob/12b2f1e86ba081390d82f1db852059777a0e1159/pkg/targets/reconciler/elasticsearchtarget/target_adapter.go#L80

Reply:
Good point.